src/com/twu/biblioteca/MainMenu.java

```diff
@@ -22,18 +22,17 @@ public void showInvalidMessage() {
     }
 
     public void manageOptions() {
-        int i = 0;
+        int counter = 0;
```

Comment:
This is better, but what is this really counting?  And if it's only possible values are 0 or 1, is it really more boolean than int?

Reply:
Ah yes, it's just checking for on or off to manage the while loop when waiting for user to provide a valid input. True or false would be cleaner.

Reply:
Love it! It's more elegant!